28.implement-str-str.py

```diff
@@ -0,0 +1,27 @@
+#
+# @lc app=leetcode id=28 lang=python3
+#
+# [28] Implement strStr()
+#
+
+# @lc code=start
+class Solution:
+    def strStr(self, haystack: str, needle: str) -> int:
+        """
+        time: 
+        O(len(needle))  if len(haystack) == len(needle)
```

Comment:
You forget for loop
O(len(haystack) * len(needle)) if len(haystack) == len(needle)
O(len(haystack)) else